eng/Versions.props

```diff
@@ -105,8 +110,13 @@
     <MicrosoftNetSdkVersion>2.0.0-alpha-20170405-2</MicrosoftNetSdkVersion>
     <MicrosoftNuGetBuildTasksVersion>0.1.0</MicrosoftNuGetBuildTasksVersion>
     <MicrosoftPortableTargetsVersion>0.1.2-dev</MicrosoftPortableTargetsVersion>
+<<<<<<< HEAD
```

Comment:
It looks like this conflict marker was left in by accident.

Reply:
I assume we want to use the versions of the dependencies specified in master-vs-deps, so I will just resolve the conflict accordingly.

Reply:
Yes, my bad. Thanks for fixing.